tasks/compass.js

```diff
@@ -13,6 +13,7 @@ module.exports = function (grunt) {
   var compass = require('./lib/compass').init(grunt);
 
   function compile(args, cb) {
+    grunt.log.verbose.writeln("Running command: " + args.join(' '));
```

Comment:
You use `grunt.log.verbose.writeln` at https://github.com/gruntjs/grunt-contrib-compass/blob/a736fed14f0e419bd54408f5143686f12f6562cd/tasks/lib/compass.js#L108 and https://github.com/gruntjs/grunt-contrib-compass/blob/a736fed14f0e419bd54408f5143686f12f6562cd/tasks/lib/compass.js#L110. Should I change those too?